apps/client/src/atoms/IntroHeading.tsx

```diff
@@ -0,0 +1,11 @@
+import { Paragraph } from "@datapunt/asc-ui";
+import React from "react";
+
+const IntroHeading = () => (
+  <Paragraph gutterBottom={8}>
+    Met de vergunningcheck kunt u zien wanneer u een omgevingsvergunning nodig
+    hebt. U kunt een vergunning nodig hebben voor:
+  </Paragraph>
+);
+
+export default IntroHeading;
```

Comment:
Why did you not place the intro-related components in `./apps/client/src/intros/shared/`?

Reply:
Done!